lib/bearerstrategy.js

```diff
@@ -294,7 +294,7 @@ Strategy.prototype.authenticate = function authenticateStrategy(req) {
       var metadataURL = self._options.identityMetadata;
 
       if (self._options.policyName) {
-        log.info('B2C: using policy %s', self._options.policyName);
+      log.info('B2C: using policy %s', self._options.policyName);
```

Comment:
As I said before, this is the signal to use the B2C flow, so this is if statement is quite large (it sets the stage for B2C, including creating a new caching key for policy). The end of the if statement is line number 312: https://github.com/AzureAD/passport-azure-ad/blob/e7305c2cfc7f6053fec9d8cbc5797791fff5c7f6/lib/bearerstrategy.js#L312